crud/common/collations.lua

```diff
@@ -0,0 +1,27 @@
+local json = require('json')
+
+local COLLATION_NAME_FN = 2
+
+local collations = {}
+
+collations.NONE = 'none'
+collations.UNICODE = 'unicode'
+collations.UNICODE_CI = 'unicode_ci'
+
+function collations.get(index_part)
+    if index_part.collation ~= nil then
+        return index_part.collation
+    end
+
+    if index_part.collation_id == nil then
+        return collations.NONE
+    end
+
+    local collation_tuple = box.space._collation:get(index_part.collation_id)
+    assert(collation_tuple ~= nil, "Unknown collation_id: " .. json.encode(index_part.collation_id))
```

Comment:
You should check is collation supported or not.

Currently it's possible to support only binary, unicode and unicode_ci.

But not all 277:
```
tarantool> #box.space._collation:select()
---
- 277
...
```

Reply:
It is checked in the code that uses collation, e.g. `select.filters`. 

I sure that it's a good approach. The `collations` module responsibility is only conversion from collcation ID to collation.